web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/EnrollEksCluster.tsx

```diff
@@ -380,7 +380,7 @@ export function EnrollEksCluster(props: AgentStepProps) {
       {showContent && (
         <>
           <Box mb={2}>
-            <Flex alignItems="center" gap={3}>
+            <Box>
```

Comment:
We have `Box` in `Box`, I think we can remove the inner one.

Reply:
Removed 👍